unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsPanelHUD/Resources/ExampleQuests.json

```diff
@@ -0,0 +1,189 @@
+[
```

Comment:
This file is going to be included with the build, I think we should store this elsewhere.

Reply:
This shouldnt be commited, gonna remove it